source/includes/k8s/file-transfer-protocol-k8s.rst

```diff
@@ -0,0 +1,167 @@
+
+- For read operations, MinIO only returns the latest version of the requested object(s) to the sftp client.
+- For write operations, MinIO applies normal versioning behavior for matching object names.
+
+Use an S3 API Client, such as the :ref:`MinIO Client <minio-client>`.
+
+
+Authentication and Access
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``SFTP`` access requires the same authentication as any other S3 client.
+MinIO supports the following authentication providers:
+
+- :ref:`MinIO IDP <minio-internal-idp>` users and their service accounts
+- :ref:`Active Directory/LDAP <minio-external-identity-management-ad-ldap>` users and their service accounts
+- :ref:`OpenID/OIDC <minio-external-identity-management-openid>` service accounts
+
+:ref:`STS <minio-security-token-service>` credentials **cannot** access buckets or objects over SFTP.
+To use STS credentials to authenticate, you must use an S3 API client or port.
+
+Authenticated users can access buckets and objects based on the :ref:`policies <minio-policy>` assigned to the user or parent user account.
+
+The SFTP protocol does not require any of the ``admin:*`` :ref:`permissions <minio-policy-mc-admin-actions>`.
+You may not perform other MinIO admin actions with SFTP.
+
+
+Prerequisites
+-------------
+
+- MinIO Operator v5.0.7 or later.
+- Enable an SFTP port (8022) for the server.
+- A port to use for the SFTP commands and a range of ports to allow the SFTP server to request to use for the data transfer.
+
+
+Procedure
+---------
+
+#. Enable SFTP for the desired Tenant:
+
+   .. tab-set::
+
+      .. tab-item:: Operator Console
+
+         - In the Operator Console, click on the tenant for which to enable SFTP.
+         - In the :guilabel:`Configuration` tab, toggle :guilabel:`SFTP` to :guilabel:`Enabled`.
+         - Click :guilabel:`Save`.
+         - Click :guilabel:`Restart` to restart MinIO and apply your changes.
+
+      .. tab-item:: Kubectl
+
+	 Use the following command to edit the Tenant YAML configuration:
+
+	 .. code-block:: yaml
+			 
+            kubectl edit tenants/my-tenant -n my-tenant-ns
+
+	 Replace ``my-tenant`` and ``my-tenant-ns`` with the desired Tenant and namespace.
+
+         In the ``features:`` section, set the value of ``enableSFTP`` to ``true``:
+
+         .. code-block:: yaml
+
+            spec:
+              configuration:
+                name: my-tenant-env-configuration
+              credsSecret:
+                name: my-tenant-secret
+              exposeServices:
+                console: true
+                minio: true
+              features:
+                enableSFTP: true
+
+         Kubectl restarts MinIO to apply the change.
+
+         You may also set ``enableSFTP`` in your Helm chart or Kustomize configuration to enable SFTP for newly created Tenants.
+
+#. If needed, configure ingress or forwarding for the SFTP port according to your local policies.
```

Comment:
Intentionally vague because _OMG ingress_. But maybe there's a useful link?

The relevant section for Operator itself is [here](https://min.io/docs/minio/kubernetes/upstream/operations/installation.html#open-the-operator-console) but that doesn't discuss anything other than Operator Console.

Reply:
@feorlen ingress is easy, I enable ingress every time I test locally with some automation scripts, if you like I can show you tomorrow. But in the meantime, you can look at some of our wikis regarding ingress in case we can put a link to them: https://github.com/minio/wiki/wiki/MinIO-Ingress-on-Kind-for-testing-and-learning

Reply:
@cniackz I would very much like to know more about your automation!! I have a VM for Operator testing but it is a very very basic setup.

Reply:
I think we are safe to just say "Ingress" for now. Anything we document will be used in production, so unless we want to potentially support that we should exercise caution :)